demo/queryparser-demo.cabal

```diff
@@ -1,21 +1,21 @@
 name:                queryparser-demo
 version:             0.1.0.0
--- synopsis:            
--- description:         
+-- synopsis:
+-- description:
 license:             MIT
 license-file:        LICENSE
 author:              Heli Wang, David Thomas, Matt Halverson
 maintainer:          heli@uber.com
--- copyright:           
+-- copyright:
 category:            Database
 build-type:          Simple
 cabal-version:       >=1.10
 
 library
   exposed-modules:     Demo
-  -- other-modules:       
-  -- other-extensions:    
-  build-depends:       base >=4.8 && <4.9
+  -- other-modules:
+  -- other-extensions:
+  build-depends:       base >=4.10
```

Comment:
will pin the base and update the PR. Thanks for getting back to me quickly.

Reply:
As I understand it, the [base package](https://hackage.haskell.org/package/base) uses package versioning policy (PVP), which has an oh-so-subtle difference from SemVer:

- PVP: Major.Major.minor.patch (e.g. 4.10.1.0)
- Semver: Major.minor.patch (e.g 5.1.0)

In both cases, major changes allow breaking changes. If Base package used SemVer, then we'd be fine using 4.x, but since it's PVP we'd need to pin to 4.10.x to avoid surprise breaking changes.